README.md

```diff
@@ -12,7 +12,7 @@ MapboxNavigation consists of two libraries. MapboxNavigation.swift which contain
 
 ## Examples
 
-We provide examples in Swift and Objective-C. Run `carthage update --platform ios` from the root folder and open MapboxNavigation.xcodeproj to try it out.
+We provide examples in Swift and Objective-C. Run `carthage update --platform ios --no-use-binaries` from the root folder and open MapboxNavigation.xcodeproj to try it out.
```

Comment:
This flag should no longer be necessary, now that I’ve renamed the binaries in MapboxDirections.swift’s latest release. Would you mind verifying that? If it still doesn’t work without this flag, I can delete the binaries; either way, this flag shouldn’t be necessary.